Tests/test_PDB_CEAligner.py

```diff
@@ -43,7 +43,7 @@ def test_cealigner(self):
         aligner.set_reference(s1)
         aligner.align(s2)
 
-        self.assertAlmostEqual(aligner.rms, 3.74, places=2)
+        self.assertAlmostEqual(aligner.rms, 3.66, places=2)
```

Comment:
Yeah, so to my point above. I'd add this optimization as an optional, so that we can keep the 'raw' CEAlign code and tests vs the 'reference' implementation in Pymol, which is what most people use, and then add tests for the optimization itself.